examples/minimal/Feast 101.ipynb

```diff
@@ -1138,7 +1105,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "KAFKA_BROKER = \"kafka:9092\""
+    "KAFKA_BROKER = os.getenv(\"DEMO_KAFKA_BROKERS\", \"kafka:9092\")"
```

Comment:
Would it make sense to hoist these variables to the top so that we know which env vars to set outside the notebook?